docs/amd_sev_snp.md

```diff
@@ -0,0 +1,36 @@
+# AMD SEV-SNP
+
```

Comment:
Please put a big warning here saying this only supported on MSHV not KVM.

Reply:
Added a warning on the top of the document